src/components/Modals/RulesModal.vue

```diff
@@ -245,22 +250,49 @@
                   <p>On each of your turns:</p>
                   <ol id="gamePlay">
                     <li>Select a card to put in the "True" or "False" playing field that was chosen at the beginning of the round, or to discard.</li>
-                    <li>At the beginning of all of the player's turns, the game will choose either the "True" or "False" playing field as the "Active" part of each program to run.
+                    <li>At the beginning of all of the player's turns, the game will choose either the "True" or "False" playing field as the "Active" part of each program to run.</li>
+                    <li>You get bonus points for good programming practice. For more information on this go to the 'Bonus Points' tab.</li>
                     <li>The game is over if a player reaches the score limit in either paths:</li>
                     <ul>
-                      <li>Short: 15 instructions</li>
-                      <li>Medium: 25 instructions</li>
-                      <li>Long: 35 instructions</li>
+                      <li>Short: 75 instructions</li>
+                      <li>Medium: 100 instructions</li>
+                      <li>Long: 125 instructions</li>
                       </ul>
                   </ol>
                 </div>
+
+                <div role="tabpanel" class="tab-pane" id="bonusPoints">
+                  <h4>Bonus Points</h4>
+                  <p>Various cards, good coding practices, or game play types will give you bonus points!  </p>
+                  <ul id="bonusPointsList">
+                    <li><h3>Master Coder - 10 points per opponent</h3></li>
+                    <p>By completing your program before an opponent has played an instruction you will gain the Master Coder bonus of 10 points per opponent.</p>
+                    <li><h3>Defensive Programmer - 15 points</h3></li>
+                    <p>Defensive Programmer is a bonus for playing all 3 safety cards</p>
+                    <li><h3>Clean System - 10 points</h3></li>
+                    <p>Clean System is a bonus for not getting any viruses by the end of the game.</p>
+                    <li><h3>Cool System - 10 points</h3></li>
+                    <p>If you don't play any overclocking cards then you will be given a bonus 10 points.</p>
```

Comment:
The game tracks if they have used an overclock at anytime? So this will make overclocking a strategic decision? That sounds like a good idea.

Reply:
Wait... did we have "overclock" as the remedy for "virus"? If so, this might be a problem.